src/components/views/elements/AppTile.tsx

```diff
@@ -406,6 +409,11 @@ export default class AppTile extends React.Component<IProps, IState> {
             ? Container.Right
             : Container.Center;
         WidgetLayoutStore.instance.moveToContainer(this.props.room, this.props.app, targetContainer);
+        if (targetContainer === Container.Right
+            && RightPanelStore.getSharedInstance().visibleRoomPanelPhase === RightPanelPhases.TimelineCard) {
+            // If the widget gets closed also close the RightPanel chat.
+            dis.dispatch({ action: Action.SetRightPanelPhase, phase: RightPanelPhases.RoomSummary });
```

Comment:
Hmm, it might be more natural to close the right panel entirely, but I'll have to think about it more.

Reply:
When testing I also found myself expecting this. It is nice though to at least see the widgets list after pressing minise on the app Tile. Otherwise there is nothing related to widgets on the screen which I would not expect when pressing a minimise button. Than a close button would be more reasonable?

Reply:
Hmm hmm, still not quite sure here. Maybe we leave it like this for now, and then adjust as we test it out more.

Reply:
This solved itself. the way I close the timeline for now (before we have per room right panel state) interferes with this. So I remove it for now and the standard behavior now is: maximised widget + right panel chat -(on widget close)-> nothing open.